src/esp/gfx/FisheyeShader.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#include "FisheyeShader.h"
+
+#include <Corrade/Utility/Assert.h>
+#include <Corrade/Utility/FormatStl.h>
+#include <Corrade/Utility/Resource.h>
+#include <Magnum/GL/Shader.h>
+#include <Magnum/GL/Texture.h>
+#include <Magnum/GL/Version.h>
+
+#include <sstream>
```

Comment:
Not needed here (sorry, I have a special sense developed for these kinds of includes :sweat_smile:)